packages/swapper/src/swappers/ThorchainSwapper/endpoints.ts

```diff
@@ -373,7 +373,7 @@ export const thorchainApi: SwapperApi = {
       }
 
       const latestOutTx = data.out_txs?.[data.out_txs.length - 1]
-      const hasOutboundTx = latestOutTx?.chain !== 'THOR'
+      const hasOutboundTx = latestOutTx !== undefined && latestOutTx.chain !== 'THOR'
```

Comment:
So turns out, if there is no `latestOutTx` it means it doesn't actually have an outbound tx 😅